### PR TITLE
implemented run_travis

### PR DESCRIPTION
**Pull request by ipa-mdl**
```
Usage: industrial_ci/scripts/run_travis [PATH] [RANGE*] [-- [ENV*]]

Parses the travis config in the given path or in the current working directory and runs all specified tests sequentially
If no range is given, the list of jobs will get printed.

The number of tests can be reduced by specifying one or more ranges:
* single job: 1 (only first)
* range: 2-3 (only second and third)
* open start, only as first range: -4 (jobs 1 to 4)
* open end, only as last range: 7- (job 7 and all following jobs)
* open range: - (all jobs)

Complex examples for a matrix wih 12 jobs:

* -4 7 8: runs jobs 1 2 3 4 7 8
* 1 7-9: runs jobs 1 7 8 9
* 1 7-9 11-: runs jobs 1 7 8 9 11 12
* -: runs all jobs

The jobs will be run in clean environments.
Only DOCKER_PORT, SSH_AUTH_SOCK, and TERM will be kept.
Additional variable names can be passed at the end.
```

It should work for most `industrial_ci` users, but I haven't tested it yet with configs that include escaped variables.

### CODE REVIEW

File: industrial_ci/scripts/run_travis [hunk 1]
```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
```
Comment (130s) [inline industrial_ci/scripts/run_travis]: Copyrights?

**Comment by ipa-mdl** [comment]
> What is this for?

It run all your travis tests locally. Like `run_ci`, but it reads the environment variables out of `.travis.yml`.

> I read the usage output but I can't come up with a good usecase

You can run all tests before pushing or if you haven't enabled tests in your fork.
Or you can just run a single test in case you want to fix a specific problem.

> Who is this for?

All users, same as for `run_ci`.

> If this is for the end-users, can you start documenting?

I will add this, there's not much to write about.
However, it would helpful to get a feedback an the usage first.

**Comment by 130s** [comment]
> It run all your travis tests locally. Like `run_ci`, but it reads the environment variables out of `.travis.yml`.

Sounds like a tremendous advantage! +1 for this addition.

**Comment by ipa-mdl** [comment]
> IIUC, the added code can be customized for the formats of other CIs.

Yes, that was the intention. However, e.g. Gitlab config is too flexible to read the variable that simple.

**Comment by ipa-mdl** [comment]
@130s: copyright header added

**Comment by 130s** [comment]
Tagged https://github.com/ros-industrial/industrial_ci/releases/tag/0.6.0

File: industrial_ci/scripts/run_travis [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+The jobs will be run in clean environments.
+Only DOCKER_PORT, SSH_AUTH_SOCK, and TERM will be kept.
+Additional variable names can be passed at the end.
+
+""" % cmd)
+
+def main(scripts_dir, argv):
+    if '--help' in argv:
+        print_help(argv[0])
```
Comment (130s) [inline industrial_ci/scripts/run_travis]: Any reason you didn't use [argparse](https://docs.python.org/3/library/argparse.html) @ipa-mdl?

Comment (ipa-mdl) [reply industrial_ci/scripts/run_travis]: It bloats the code and is overkill for just printing the usage.

**Pull request merged**